plugins/source/aws/resources/services/cloudwatchlogs/subscription_filters.go

```diff
@@ -0,0 +1,47 @@
+package cloudwatchlogs
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func subscriptionFilters() *schema.Table {
+	return &schema.Table{
+		Name:        "aws_cloudwatchlogs_log_group_subscription_filters",
+		Description: `https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_SubscriptionFilter.html`,
+		Resolver:    fetchCloudwatchlogsSubscriptionFilters,
+		Multiplex:   client.ServiceAccountRegionMultiplexer("logs"),
+		Transform:   transformers.TransformWithStruct(&types.SubscriptionFilter{}, transformers.WithPrimaryKeys("FilterName", "CreationTime")),
+		Columns: []schema.Column{
+			client.DefaultAccountIDColumn(false),
+			client.DefaultRegionColumn(false),
+			{
+				Name:        "log_group_arn",
+				Description: "The Amazon Resource Name (ARN) of the log group.",
+				Type:        schema.TypeString,
+				Resolver:    schema.ParentColumnResolver("arn"),
+			},
+		},
+	}
+}
+func fetchCloudwatchlogsSubscriptionFilters(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	config := cloudwatchlogs.DescribeSubscriptionFiltersInput{
+		LogGroupName: parent.Item.(types.LogGroup).LogGroupName,
```

Comment:
Having this here adds an extra call per log group. If the API allows loggroupless query (I'm inventing new words today) it would be better to do it that way to save on api calls, as with the metric filters. In that case the resource would need to be moved to the top level.

Reply:
It doesn't... That line had gotten removed while cleaning up the PR

Reply:
![Screenshot 2023-02-14 at 10 15 28 AM](https://user-images.githubusercontent.com/30294676/218794791-96020182-20f6-4c11-ac62-d0c6e8ed375b.png)